### ISSUE TITLE
Broken functionality on update to 3.0.5

### ISSUE THREAD

**Issue by andaniel05** [opis/closure#12]
```php
<?php

require_once 'vendor/autoload.php';

use function Opis\Closure\{serialize as s, unserialize as u};

$obj = new \stdClass;
$obj->closure = function () use ($obj) {};

$text = s($obj);

// Error on update to opis 3.0.5 from 3.0.4.
// Success in older versions.
u($text);

**Comment by andaniel05** [comment]
My php version is 7.1.1

**Comment by msarca** [comment]
Fixed in 3.0.6
